File: ui/currency-converter-ui/src/models/currency.ts
export class Currency {
    id: number;
    code: string;
    displayName: string;
    description: string;
    symbol: string;

    constructor(id: number, code: string, displayName: string, description: string, symbol: string) {
        this.id = id;
        this.code = code;
        this.displayName = displayName;
        this.description = description;
        this.symbol = symbol;
    }

    getDisplay(): string {
        return this.code + ' - ' + this.displayName;
    }
}

File: api/CurrencyConverterAPI/Entity/Currency.cs
namespace CurrencyConverterAPI.Entity
{
    public class Currency
    {
        public int CurrencyId { get; set; }
        public string Code { get; set; }
        public string DisplayName { get; set; }
        public string Description { get; set; }
        public char Symbol { get; set; }
    }
}
